src/Objects/_PVector.js

```diff
@@ -0,0 +1,283 @@
+module.exports = function(options, undef) {
+  'use strict';
+
+  const TAU = 2 * Math.PI,
+        //TAU = options.PConstants.TAU,
+        lerp = function(start, stop, amt) {
+          return amt * (stop - start) + start;
+        },
+        sq = function(n) {
+          return n*n;
+        },
+        argsErr = function(mtd, len, min) {
+          throw 'Too few args passed to ' + mtd +
+                '() [' + len + ' < ' + min + '].';
+        };
+
+  function _PVector(x, y, z) {
+    this.x = +x || 0;
+    this.y = +y || 0;
+    this.z = +z || 0;
+  }
+
+  _PVector.fromAngle = function(angle, target) {
+    return target? target.set(Math.cos(angle), Math.sin(angle))
+                 : new _PVector(Math.cos(angle), Math.sin(angle));
+  };
+
+  _PVector.random2D = function(target, parent) {
+    //const rnd = parent? parent : target instanceof Processing? target : Math;
+    const rnd = parent? parent : target && target.random? target : Math;
```

Comment:
Yes, they will compute things differently:
https://github.com/processing-js/processing-js/blob/master/src/P5Functions/Math.js#L428

Both Java Mode & PJS have their own **random()** function holding state for the **randomSeed()**:
https://processing.org/reference/randomSeed_.html

Besides, **random2D()** got 4 overloaded signatures:
https://processing.org/reference/PVector_random2D_.html

1. random2D()
2. random2D(parent)
3. random2D(target)
4. random2D(target, parent)